### PR TITLE
organizing less files

### PR DESCRIPTION
**Pull request by gvn**
An initial stab at LESS organization.

This puts LESS files parallel to their target view or component and names them with the same prefix.

My thought is that the `components` directory contains global/re-usable components and the `pages` (rename to `views`?) directory contains all view-specific LESS.

### CODE REVIEW

**Comment by gvn** [comment]
How does this feel @thisandagain and @k88hudson ?

File: www_src/main.less [hunk 1]
```diff
@@ -1,6 +1,17 @@
 @import "normalize";
-@import "project-map";
-@import "positionable";
+
+// Components
+
+@import "components/link/link.less";
```
Comment (k88hudson) [inline www_src/main.less]: .less not needed

Comment (gvn) [inline www_src/main.less]: Whoops...autocomplete.

**Comment by k88hudson** [comment]
Looks good to me, a great improvement. A couple of notes

* We may want to have different entry point less files for each page eventually. I think @thisandagain was thinking that would be a better idea.  If we run into really big file sizes or something we could reconsider, but personally, I think your current approach simpler and works fine.
* There are a few more moving pieces/things that might be better automated (adding the right ID to the container div, adding the import statement), but happy to see those filed and done as follow-ups.

**Comment by gvn** [comment]
Yeah, I was thinking I could document the standards once we agree on them, and then we can automate later.

**Comment by k88hudson** [comment]
@gvn awesome, thanks for getting this done

**Comment by thisandagain** [comment]
This is awesome. Thanks for the quick turnaround @gvn ! :panda_face:

**Comment by gvn** [comment]
@Pomax FYI – Might wanna do a rebase first thing tomorrow if you have any patches you're working on!

**Pull request merged**